src/ProjectSystemDogfoodSetup/ProjectSystemDogfoodSetup.csproj

```diff
@@ -17,6 +17,7 @@
     <IncludeDebugSymbolsInVSIXContainer>false</IncludeDebugSymbolsInVSIXContainer>
     <IncludeDebugSymbolsInLocalVSIXDeployment>false</IncludeDebugSymbolsInLocalVSIXDeployment>
     <ImportVSSDKTargets>true</ImportVSSDKTargets>
+    <DeployExtension Condition="'$(DeployVsixExtension)' == 'false'">false</DeployExtension>
     <RestorePackages>true</RestorePackages>
```

Comment:
Setting `DeployExtension` to true makes msbuild think the non-VS projects(MS.VS.PS.Managed in this case) also to be vsix projects. It looks for vsixmanifest and fails. Although we specifically mention not to import VSSDK targets, setting the DeplyExtension property imports the VS SDK targets and eventually results in failure